lib/spack/spack/build_systems/autotools.py

```diff
@@ -184,12 +189,6 @@ def patch(self):
             ``config.guess``
         """
```

Comment:
@lee218llnl Can we remove the empty `patch` function? Or was it left there for same reason?